.github/dependabot.yml

```diff
@@ -11,12 +11,12 @@ updates:
       interval: "weekly"
 
   - package-ecosystem: "gomod"
-    # Look for `go.mod` file in the `root` directory
-    directory: "/"
-    # Check the gomod registry for updates every Monday
-    schedule:
-      interval: "weekly"
-    reviewers:
-      - "platform"
-    # max number of pull requests that dependabot will open in tandem
-    open-pull-requests-limit: 4
+      # Look for `go.mod` file in the `root` directory
+      directory: "/"
+      # Check the gomod registry for updates every Monday
+      schedule:
+        interval: "weekly"
+      reviewers:
+        - "platform-squad"
```

Comment:
Hi Sredny, while investigating the GitHub team name change, I noticed that there was an error with the team name, and that the org should be included:
- Error: https://github.com/TykTechnologies/tyk-pump/pull/472#issuecomment-1215162209 
- Docs for team name: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#reviewers
- Jira ticket for bug: https://tyktech.atlassian.net/browse/TD-1168
From the docs:
> You must use the full team name, including the organization

Reply:
thanks for the head up!!! I will update properly

Reply:
fixed